test/VestingWalletWithCliffAndClawback.t.sol

```diff
@@ -175,12 +175,12 @@ contract VestingWalletWithCliffAndClawbackTest is Test {
         _assertReleasableIsAmount(0);
     }
 
-    function testCannotReleaseDuringCliff() public {
+    function testNothingReleasedBeforeCliff() public {
         skip(startDelay);
-        _assertAbilityToRelease(false);
+        _assertReleasableIsAmount(0);
```

Comment:
After this change, the `_assertAbilityToRelease` helper is no longer used and can be removed.

Reply:
https://github.com/gerrrg/VestingWalletWithCliffAndClawback/blob/4b8176952a4e9308c5cd974259a0a9e55229d9a8/test/VestingWalletWithCliffAndClawback.t.sol#L200